docs/connector-development/connector-builder-ui/incremental-sync.md

```diff
@@ -0,0 +1,133 @@
+  "sectionId": "business",
+  "sectionName": "Business",
+  "webPublicationDate": "2023-04-15T07:30:58Z",
+}
+```
+
+Then when a sync is triggered for the same connection the next day, the following request is made:
+```
+curl 'https://content.guardianapis.com/search?order-by=oldest&from-date=2023-04-15T07:30:58Z&to-date=2023-04-16T10:18:08Z'
+```
+
+The `from-date` is set to the cutoff date of articles synced already and the `to-date` is set to the new "now".
+
+## Advanced settings
+
+The description above is sufficient for a lot of APIs. However there are some more subtle configurations which sometimes become relevant. 
+
+### Step
+
+When incremental syncs are enabled, the connector is not fetching all records since the cutoff date at once - instead it's splitting up the time range between the cutoff date and the desired end date into intervals based on the "Step" configuration (by default it's set to 1 month)
+
+For example if the "Step" is set to 10 days (`P10D`) for the Guardian articles stream described above and a longer time range, then the following requests will be performed:
+```
+
+curl 'https://content.guardianapis.com/search?order-by=oldest&from-date=2023-01-01T00:00:00Z&to-date=2023-01-10T00:00:00Z'
+
+curl 'https://content.guardianapis.com/search?order-by=oldest&from-date=2023-01-10T00:00:00Z&to-date=2023-01-20T00:00:00Z'
+
+curl 'https://content.guardianapis.com/search?order-by=oldest&from-date=2023-01-20T00:00:00Z&to-date=2023-01-30T00:00:00Z'
+
+...
+```
+
+After an interval is processed, the cursor value of the last record will be saved as part of the connection as the new cutoff date.
+
+In most cases, the default step size is fine, there are two reasons to change it:
+* **The API is unreliable** and the cutoff date should be saved more often to prevent resync of a lot of records - if the "Step" size is a day, then at most one day worth of data needs to be resync in case the sync fails halfway through. However, a smaller step size might cause more requests to the API and more load on the system.
+* **The API requires the connector to fetch data in pre-specified chunks** - for example the [Exchange Rates API](https://exchangeratesapi.io/documentation/) makes the date to fetch data for part of the URL path and only allows to fetch data for a single day at a time
+
+### Lookback window
+
+The "Lookback window" specifies a duration that is subtracted from the last cutoff date before starting to sync.
+
+Same APIs update records over time but do not allow to filter or search by modification date, only by creation date. For example the API of The Guardian might change the title of an article after it got published, but the `webPublicationDate` still shows the original date the article got published initially.
+
+In these cases, there are two options:
+* **Do not use incremental sync** and always sync the full set of records to always have a consistent state - depending on the amount of data this might not be feasable 
```

Comment:
nit: this is also undesirable for some use cases even if the amount of data is small. This prevents the user from keeping a history of the changes (dedup + history)